### ISSUE TITLE
makeglossaries

### ISSUE THREAD

**Issue by RemDelaporteMathurin** [xu-cheng/latex-action#96]
Hi @xu-cheng 

I would like to know how to use this action to also make the glossary in a document?

See https://github.com/dante-ev/latex-action/issues/22

**Comment by xu-cheng** [comment]
Please see https://tex.stackexchange.com/questions/1226/how-to-make-latexmk-use-makeglossaries and http://mirrors.ctan.org/support/latexmk/example_rcfiles/glossaries_latexmkrc

**Comment by RemDelaporteMathurin** [comment]
Thanks for your reply! 

Just to make it clear, I just have to add a .latexmkrc file with the commands detailed in the link to the root of the directory?
And then I can use the action as is?

**Comment by xu-cheng** [comment]
Yes. You can also test it locally by running `latexmk -pdf main.tex`
